config/locales/en/activerecord.yml

```
@@ -151,6 +151,9 @@ en:
       votation_type:
         one: Votation type
         other: Votation types
+      cookies_vendor:
```

There's a typo here, meaning this code isn't doing anything :thinking:. Same in Spanish.

```suggestion
      cookies/vendor:
```

I was wondering whether we should change the `  <%= f.submit t("shared.save") %>` line to  `<%= f.submit %>` in `Admin::Cookies::Vendors::FormComponent`. That way we'd actually use this code :thinking:. I guess we could also use it in some other places, but I'm not sure it's worth it :thinking:.